### DISCUSSION TITLE
Is there a way to disable the client side router?

### DISCUSSION THREAD

**Discussion by nCrafts** [sveltejs/kit#9381] [Q&A]
I am trying to reduce the number of http requests on my page. I have a particular page on my app that would use a lot of components but does not need the router at all. I know this feature was discussed before [here](https://github.com/sveltejs/kit/issues/231)

Is there a way to do this now? Disable client side routing, without disabling client side rendering?

**Comment by vekunz** [comment]
Maybe `data-sveltekit-reload` is an option for you https://kit.svelte.dev/docs/link-options#data-sveltekit-reload

**Reply by nCrafts** [reply]
I believe this changes the behaviour of clicking on links and pre-loading data. What I'd like to do is essentially not load router components on a particular page. How the users visit this page is not relevant, since it will always be loaded directly in the browser via a URL.

**Reply by kevbook** [reply]
I have the same question - when I do `csr = false` then component interactivity turns off as well. I want the components to interact but routing is enabled server-side only.

**Reply by Fasteroid** [reply]
Same question, will this recieve any attention?

**Reply by einarpersson** [reply]
So `csr = false` does not just mean "turn off client side routing" but rather "no javascript"? If that is the case, the name is rather confusing...

**Reply by sutanAE** [reply]
have you tried adding `rel="external"` to every `<a>` tag?
